feature/tabs/src/main/java/com/epmedu/animeal/tabs/presentation/TabsScreen.kt

```diff
@@ -97,9 +97,6 @@ private fun BottomNavigationBar(
                             contentDescription = stringResource(item.title)
                         )
                     },
-                    label = {
-                        Text(text = stringResource(id = item.title))
```

Comment:
What about `title` field inside `NavigationTab`? Is it use now?

Reply:
It no longer has any use.

Reply:
please rename `title` to `contentDescription`

Reply:
Changes done